content/patterns/checkbox/examples/checkbox-mixed.html

```diff
@@ -18,103 +18,104 @@
     <script src="js/checkbox-mixed.js"></script>
   </head>
   <body>
-    <nav aria-label="Related Links" class="feedback">
+    <nav aria-label="関連リンク" class="feedback">
       <ul>
-        <li><a href="https://github.com/w3c/aria-practices/projects/9">Related Issues</a></li>
-        <li><a href="../checkbox-pattern.html">Design Pattern</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/projects/9">関連する Issues</a></li>
+        <li><a href="../checkbox-pattern.html">デザインパターン</a></li>
       </ul>
     </nav>
     <main>
-      <h1>Checkbox Example (Mixed-State)</h1>
+      <h1>チェックボックスの例（混合状態）</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/962f6e771dccd4e3bbf8b83da717fb8d75b861da -->
 
       <section>
-        <h2>About This Example</h2>
+        <h2>この例について</h2>
         <img alt="" class="example-page-example-icon" src="../../../images/pattern-checkbox.svg">
         <p>
-          This example demonstrates using the <a href="../checkbox-pattern.html">Checkbox Pattern</a> to create a tri-state, or mixed-state, checkbox.
-          In this implementation, the mixed-state checkbox represents the state of a set of standard HTML checkboxes.
-          If none of the checkboxes in the set are checked, the mixed state checkbox is not checked, and if all members of the set are checked, the mixed state checkbox is checked.
-          If the set contains both some checked and unchecked checkboxes, the mixed state checkbox is partially checked.
-          Activating the tri-state checkbox changes the states of the checkboxes in the set.
+          この例では、<a href="../checkbox-pattern.html">チェックボックスのパターン</a>を使用して、三値状態、又は混合状態のチェックボックスを作成する方法を示します。
+          この実装では、混合状態のチェックボックスは、一連の標準 HTML チェックボックスの状態を表します。
```

Comment:
「一連の」は説明されればわかるが初見だと理解が難しそう。。
後ろでセットと言っているし、セットに揃えたほうがよさそう。そして、「複数の」という情報が失われているので補う。

```suggestion
          この実装では、混合状態のチェックボックスは、複数の標準 HTML チェックボックスのセットの状態を表します。
```

「の」が多いが、下手に分けるよりもいい気がする